showpass/Showpass-E2E-tests
# TA-18 Delete old and add a new payment method

**Description:**
Due to a known Cypress limitation any scripts involving iFrames have to be set up on a case by case basis. Unfortunately, we weren't able to find a proper workaround for the populateCardInformationForm() method before. With that said, another method [cy.populateBillingAddressForm(this.testdata.userAddress)] is working just fine but because two forms need to be populated before a new payment method can be submitted.
In this PR we will be removing an old and adding a new credit card payment.

Where the method can be found:
cypress/support/page-methods/user-settings-methods.js

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-18

**How to test:**
Run the following test suite and ensure corresponding specs pass.
cypress/e2e/user-settings/profile.cy.js

**Test execution results:**

<img width="1003" alt="TA-18-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/6314d740-87f5-4ac8-8b1e-53b3654126b2">
<img width="1717" alt="TA-18-test-suite-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/45d1ccd4-2d83-44bf-8ab0-6b278982d451">
